app/models/ingest_job.rb

```diff
@@ -628,6 +628,7 @@ def generate_success_email_array
         schema_url = 'https://singlecell.zendesk.com/hc/en-us/articles/360061006411-Metadata-Convention'
         message << "This metadata file was validated against the latest <a href='#{schema_url}'>Metadata Convention</a>"
         message << "Convention version: <strong>#{project_name}/#{current_schema_version}</strong>"
+        message << "Group-type metadata columns with more than 200 unique values are not stored (ie. unavailable for visualization)."
```

Comment:
All prominent sources and [manuals of style](https://www.google.com/search?q=manual+of+style+ie) I know use a period between each letter.  I.e.:

```suggestion
        message << "Group-type metadata columns with more than 200 unique values are not stored (i.e. unavailable for visualization)."
```

Reply:
I don't think users are going to know/care what 'stored' means in this context.  
```suggestion
        message << "Group-type metadata columns with more than 200 unique values are not made available for visualization."
```